cmd/nodecmd/create.go

```diff
@@ -658,6 +657,7 @@ func createNodes(cmd *cobra.Command, args []string) error {
 	}
 	startTime := time.Now()
 	if addMonitoring {
+		spinSession := ux.NewUserSpinner()
```

Comment:
https://github.com/ava-labs/avalanche-cli/pull/2522/files#diff-af9a7cff84db7d7f438c72aad8de4aeb4f53e53bcd605dbb2a0387eac8e57d95R652 is already there. not sure why we need another local one